src/inmanta/deploy/timers.py

```diff
@@ -33,6 +37,14 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class TimerState(enum.Enum):
+
+    IDLE = 0
+    WAITING = 1
+    DEPLOY_REQUESTED = 2
+    REPAIR_REQUESTED = 3
```

Comment:
This isn't used afaics

Reply:
thx, I had started working on a way to handle priority bumping when the repair interval is shorter than the deploy interval, but it was just too complicated to be worth it.

Reply:
Just call `deploy_resource()` again and it should get magically taken care of?

Reply:
The issue is: call it when and with which parameters. 

when assuming the shortest timer has the highest priority, everything is simple: only the shortest one matters.

If this is not true, the complication is what state are we in?  Did we already schedule the shortest one or not? 
1. So either we run two timers (which has its own complications)
2. We keep a state machine to determine if we are past the shortest (but depending on the config and resource state, we traverse it differently, which is complicated)
3. We work based of time (are we past the shortest time now), which forces one to consider the non-monotonous nature of datetime

For a case that is not expected to happen (and where we can tell the user DUH!), I'm not going to invite this complexity in

Reply:
Makes sense. Now that we document that deploy is supposed to be shorter, this is a moot point.